cpp/inc/bond/core/transforms.h

```diff
@@ -550,9 +533,12 @@ class To
     template <typename X>
     bool Base(const X& value) const
     {
-        bool done = AssignToBase(_var, value);
-        Validator::Base(done);
-        return done;
+        if (bool done = AssignToBase(_var, value))
+        {
+            UnexpectedStructStopException();
+        }
+
+        return false;
```

Comment:
Just wanted to check that it's structured this way on purpose (e.g., to give a name to the return value of `AssignToBase`) and that it shouldn't read `return done`.